wallet/src/Cardano/Wallet/Kernel/Transactions.hs

```diff
@@ -177,7 +178,7 @@ pay activeWallet spendingPassword opts accountId payees = do
              Left e      -> return (Left $ PaymentNewTransactionError e)
              Right (txAux, partialMeta, _utxo) -> do
                  let sz = fromIntegral $ BL.length $ serialize txAux
-                 maxSz <- Node.getMaxTxSize (walletPassive activeWallet ^. walletNode)
+                 maxSz <- toBytes <$> Node.getMaxTxSize (walletPassive activeWallet ^. walletNode)
```

Comment:
Error message was incorrectly showing the value as `Byte xxx bytes.` instead of just `xxx bytes`